ompi/mca/osc/ucx/osc_ucx_active_target.c

```diff
@@ -256,6 +256,9 @@ int ompi_osc_ucx_post(struct ompi_group_t *group, int assert, struct ompi_win_t
                                                  myrank + 1, &result, sizeof(result),
                                                  remote_addr);
                 if (ret != OMPI_SUCCESS) {
+                    free(ranks_in_grp);
+                    free(ranks_in_win_grp);
+                    ompi_group_free(&win_group);
```

Comment:
Is this a verification path?
Does ompi_comm_group allocate a group? 
Just to make sure that it isn't a pointer to an internal object.

Reply:
yes, it allocates it in line 226, it's released at proper function exit, line 284